src/Persistence/Array_.php

```diff
@@ -110,17 +108,15 @@ public function load(Model $m, $id, $table = null)
      * @param Model  $m
      * @param mixed  $id
      * @param string $table
-     *
-     * @return array|false
      */
-    public function tryLoad(Model $m, $id, $table = null)
+    public function tryLoad(Model $m, $id, $table = null): ?array
```

Comment:
Looks like $table can be only string, so we can set type here. I doubt that somebody will set table name as integer.